apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/handlers/NovaErrorHandler.java

```diff
@@ -97,6 +97,13 @@ else if (message.indexOf("already exists") != -1)
                exception = new ResourceNotFoundException(message, exception);
             }
             break;
+        case 500:
```

Comment:
Can you add this comment to the code. It seems to me like somewhat inconsistent openstack behavior.